src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -25,20 +25,30 @@ public static void Bind(this IConfiguration configuration, object instance)
             }
         }
 
-        public static T Get<T>(this IConfiguration configuration)
+        public static T Get<T>(this IConfiguration configuration, T defaultValue)
```

Comment:
I don't want to make a distinction. I would like something that is neutral. I think `T` and `@default` are neutral. `TResult` and `defaultResult` are neutral too, but "result" conveys things that don't completely fit in here.